### PR TITLE
Updated documentation and Csv

### PR DESCRIPTION
**Pull request by MirzaBaig715**
@sadanarshad Added documentation and correct errors.File Handling with CSV.

### CODE REVIEW

File: weatherman.py [hunk 1]
```diff
@@ -1,214 +1,362 @@
...
+
+# MAIN PROGRAM
+
+report_type = str(input("For Annual Report :1\n"
+                        "For Hottest Days Report:2\n"
+                        "Enter Report Type: "
+                        )
+                  )
+# Take input from user. What type of report to display
+path = "*.txt"
```
Comment (sadan) [inline weatherman.py]: What is this? This is not complete path of the weather files i guess

File: weatherman.py [hunk 2]
```diff
@@ -1,214 +1,362 @@
...
+    for file in files:
+        # iterate over all files
+        lines = []
+        L = []
+        with open(file) as f:
+            # Skip the first two lines
+            f.__next__()
+            f.__next__()
+            # Read all from a file and store it in List @lines
+            lines = csv.reader(f)
```
Comment (sadan) [inline weatherman.py]: use CSV dict reader to extract data from files. That will refactor and improve your logic more.

File: weatherman.py [hunk 3]
```diff
@@ -1,214 +1,362 @@
...
+
+
+def annual_report():
+    """
+    It will show the annual report to the user
+
+    """
+
+    print(str("Year").rjust(7),
+          str("MAXTemp").rjust(7),
```
Comment (sadan) [inline weatherman.py]: this is good.

**Review by sadan — CHANGES_REQUESTED** [review]

File: weatherman.py (hunk 3 above)
Comment (MirzaBaig715) [reply weatherman.py]: That's Great ! Thanks

File: weatherman.py (hunk 2 above)
Comment (MirzaBaig715) [reply weatherman.py]: Yes i will extract using DictReader. Will also make it more memory efficient. Thanks for reviewing.

File: weatherman.py (hunk 1 above)
Comment (MirzaBaig715) [reply weatherman.py]: files are in current Project Directory. We just have to extract _*.txt_ files from folder.

File: weatherman.py [hunk 4]
```diff
@@ -1,214 +1,206 @@
...
+
+
+def main_function():
+    """
+    This is Main function
+
+    """
+    files = glob.glob("*.txt")
+    for file in files:
+        MAXTemp = []
```
Comment (sadan) [inline weatherman.py]: variable name are always written in small letters **max_temp**, unless some sort of global or settings variable written in all caps.

**Review by sadan — APPROVED** [review]

**Pull request merged**